src/pages/LandingPage.tsx

```diff
@@ -17,14 +19,17 @@ import { SearchInput } from '@patternfly/react-core';
 import ThIcon from '@patternfly/react-icons/dist/js/icons/th-icon';
 import ThListIcon from '@patternfly/react-icons/dist/js/icons/th-list-icon';
 import {Helmet} from 'react-helmet';
+import { TableComposable, Thead, Tr, Th, Tbody} from '@patternfly/react-table';
 
 import {SidebarTags} from "../components/SideBar/SidebarTags";
 import {NoMatchFound} from "../components/NoMatchFound/NoMatchFound";
 import {usePaginatedGallery} from "../components/Card/usePaginatedGallery";
 import {GalleryTemplate} from "./GalleryTemplate";
+import {ListView} from './ListView';
 
 export const LandingPage: FunctionComponent = () => {
   const [searchInput, setSearchInput] = useState('');
+  const [view, setView] = useState('grid')
```

Comment:
Lets add some type safety over here:
```suggestion
  const [view, setView] = useState<'grid'|'list'>('grid')
```

This will prevent stuff like : `setView('explode')` or hard to spot typos (depending on your font :grin: ) like `setView('Iist')`, `setView('grld')`